charts/victoria-metrics-anomaly/Chart.yaml

```diff
@@ -3,7 +3,7 @@ type: application
 name: victoria-metrics-anomaly
 description: Victoria Metrics Anomaly Detection - a service that continuously scans Victoria Metrics time series and detects unexpected changes within data patterns in real-time.
 version: 1.4.2
-appVersion: v1.15.6
+appVersion: v1.15.9
```

Comment:
Could you also add a changelog entry for `appVersion` change?

Reply:
done